### PR TITLE
LaTeX: Skills and Resources Audit

### PR DESCRIPTION
**Pull request by Joel-Milligan**
Transferred Jayden's Skills and Resources Audit into LaTeX.

### CODE REVIEW

File: docs/SkillsResourcesAudit/Skills and Resources Audit.tex [hunk 1]
```diff
@@ -0,0 +1,62 @@
+\documentclass[a4paper]{article}
+\usepackage{array}
+\usepackage{makecell}
+
+\title{Skills and Resources Audit}
+\author{Jayden Teo}
```
Comment (APM246) [inline docs/SkillsResourcesAudit/Skills and Resources Audit.tex]: I did the Skills/Resources audit. Jayden did the Risk Register

**Review by APM246 — CHANGES_REQUESTED** [review]

**Review by APM246 — APPROVED** [review]

File: docs/SkillsResourcesAudit/Skills and Resources Audit.tex (hunk 1 above)
Comment (Joel-Milligan) [reply docs/SkillsResourcesAudit/Skills and Resources Audit.tex]: Sorry about that, got my authors mixed up while converting all the doccos to LaTeX

Comment (APM246) [reply docs/SkillsResourcesAudit/Skills and Resources Audit.tex]: All good

**Comment by APM246** [comment]
The PDF should be regenerated to reflect changes to the .tex file @Joel-Milligan

**Pull request merged**